src/AbstractSQLRules2SQL.ts

```diff
@@ -939,6 +940,12 @@ const typeRules: Dictionary<MatchFn> = {
 		const date = DateValue(getAbstractSqlQuery(args, 0), indent);
 		return `DATE(${date})`;
 	},
+	DateTrunc: (args, indent) => {
+		checkArgs('DateTrunc', args, 2);
+		const field = Field(getAbstractSqlQuery(args, 0), indent);
+		const str = args[1];
+		return `date_trunc('${str}',${field})`;
```

Comment:
To match the format of the rest of the generated sql:
```suggestion
		return `DATE_TRUNC('${str}', ${field})`;
```